packages/boxed-expression-component/src/contextMenu/MenuWithHelp/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from "./MenuItemWithHelp";
```

Comment:
is this file really necessary? I mean, you can remove this and adjust the imports to import directly...

Reply:
sorry, I am new to react, I see this is node in any folder that represents some 'module/package' with standalone logic, could you please provide an example in existing codebase how to do it to align with your thoughts?